src/Symfony/Component/OptionsResolver/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.3
+---
+
+ * Add `OptionsResolver::ignoreUndefined()` to ignore not defined options while resolving
```

Comment:
It should be `setIgnoreUndefined` after the last changes, and the we should add a log for the `OptionConfigurator` too

Reply:
yes, I was on it.

Reply:
https://github.com/symfony/symfony/pull/50305